### DISCUSSION TITLE
Is there a way to download only a table

### DISCUSSION THREAD

**Discussion by aborruso** [adbar/trafilatura#177] [Q&A]
First of all, thank you, this is a great project.

Hi,
if I run

```
trafilatura  --URL "https://www.agenas.gov.it/covid19/web/index.php?r=site%2Ftab1"
```

I have a text output, in which I have also the content of a table. Is there a way to download only the table?

Thank you

**Comment by adbar** [comment]
Hi @aborruso, sorry for the late answer!
Maybe you could use XML output and then the `<table>` tag in order to get the table alone.

**Reply by aborruso** [reply]
Thank you.

I have tried to use for the URL below, but I do not have all the cells, but only the heading.

Is there a way to force and wait the full download?

```
trafilatura -out xml -u "https://www.agenas.gov.it/covid19/web/index.php?r=site%2Ftab2"
```

**Reply by adbar** [reply]
I'm not sure how to do this. Maybe you could use another download utility which renders JavaScript and then pipe the result to Trafilatura.
